server/utils/recipeUtils.js

```diff
@@ -50,6 +50,20 @@ const removeRecipe = asyncHandler(async (recipeID, Recipe) => {
 	})
 })
 
+const emptyRecipe = asyncHandler(async (Recipe) => {
+	await Recipe.sync()
+
+	return await Recipe.destroy({
+		where: {},
```

Comment:
Ok so this might not work but have you tried removing the where clause? I've done queries without it so I'm wondering if the fact that your using it with an empty object is causing some errors?

Reply:
Oh and I just realized something that'll probably work. You can use raw queries
```javascript
await sequelize.query("DELETE FROM Recipes;");
```
This is probably the easy way of doing it

Reply:
I will push the fix for this